python/paddle/v2/fluid/initializer.py

```diff
@@ -14,14 +14,37 @@
 
 import framework
 import numpy as np
+import contextlib
 
 __all__ = [
-    'Constant',
-    'Uniform',
-    'Normal',
-    'Xavier',
+    'Constant', 'Uniform', 'Normal', 'Xavier', 'force_init_on_cpu',
+    'init_on_cpu'
 ]
 
+_force_init_on_cpu_ = False
```

Comment:
Since it does only wok on Constant Initializer, let's make it more accurate.
```
_force_constant_init_on_cpu_
```

Reply:
I find that this flag will globally effect `fill_constant` operator. So,  maybe 

```
_force_fill_constant_on_cpu_
```
is more appropriate